node/risk-admin/server/routes/shp2json.js

```diff
@@ -23,7 +21,7 @@ module.exports = {
       const { data: geojson } = await ogr2ogr(zipfile)
 
       // uncomment the below to use dummy data to bypass having to upload an actual shape file on dev
-      // const geojson = exampleJson
+      // const geojson = require('./dummy-data/example_file.json')
```

Comment:
I like this, didn't think of just adding the dummy data on the commented line constant!